tools/wheel_setup.py

```diff
@@ -28,7 +28,7 @@
     'Programming Language :: Python :: 3'
 ]
 KEYWORDS = 'science math physics optimization ai evolutionary-computing parallel-computing metaheuristics'
-INSTALL_REQUIRES = ['numpy', 'cloudpickle']
+INSTALL_REQUIRES = ['numpy', 'cloudpickle', 'scipy']
```

Comment:
scipy should be removed, because this file lists only the mandatory dependencies.

Reply:
Fixed in 8acd5c5